package.json

```diff
@@ -274,6 +274,7 @@
     "@react-aria/menu": "3.10.0",
     "@react-aria/overlays": "3.15.0",
     "@react-aria/utils": "3.18.0",
+    "@react-awesome-query-builder/core": "6.4.1",
```

Comment:
Without this, we cannot see the type details of the `@react-awesome-query-builder` package. I am not sure whether there is a better way to have those types visible in the IDE.